### DISCUSSION TITLE
ImageFolder or its alternative

### DISCUSSION THREAD

**Discussion by eagle-k** [xamarin/TorchSharp#994] [Q&A]
Is there an equivalent class for [ImageFolder](https://pytorch.org/vision/main/generated/torchvision.datasets.ImageFolder.html)?
Or is there a way to create an equivalent dataset and dataloader?

**Comment by NiklasGustafsson** [comment]
No, but it would be a great contribution from someone willing to put a little time into it.

**Comment by GeorgeS2019** [comment]
https://github.com/pytorch/data/blob/main/examples/vision/imagefolder.py
